docs/faq.rst

```diff
@@ -20,3 +20,19 @@ A: No. ``@most/core`` is focused on reactive event programming rather than Array
 **Q: I want to stream chunked data with flow control (also known as "back pressure"). Should I use ``@most/core`` for that?**
 
 A: No. ``@most/core`` is focused on reactive event programming rather than chunked/block data streaming :ref:`Read more here <when should you use it>`.
+
+**Q: Are ``@most/core`` event streams the same as Node streams?**
+
+A: No. ``@most/core`` and Node streams are fairly different things, each of which happened to choose the name "stream" for one of its core concepts. They have a different primary goals, and thus different architectures and APIs.
+
+``@most/core`` :ref:`Event Streams`:
+
+* Deal with discrete events as as they happen, such as mouse clicks, where reactivity (timeliness) is a major factor
```

Comment:
```
 ``@most/core`` :ref:`Event Streams`

 * deal with discrete events as they happen, such as mouse clicks, where reactivity (timeliness) is a significant factor;
 * provide an API of functions for filtering, transforming, merging, etc. discrete event streams; and
 * add new functionality by writing new event sources and functions.
```

Applying this removes an extra _as_, swaps the word _major_ with _significant_, and sets the vertical list that complete a sentence as per the Chicago Manual of Style.

Reply:
Thanks! Done in e1ff33b.